pkg/provisioner/block/block.go

```diff
@@ -163,10 +163,15 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions, ad *i
 
 	glog.Infof("Creating volume size=%v AD=%s compartmentOCID=%q", volSizeMB, *ad.Name, block.client.CompartmentOCID())
 
+	prefix := strings.TrimSpace(os.Getenv(volumePrefixEnvVarName))
+	if prefix != "" && !strings.HasSuffix(prefix, "-") {
+		prefix = fmt.Sprintf("%s%s", prefix, "-")
```

Comment:
I would rename this to prefixedName or sth similar

Reply:
Since this is only the prefix and not the prefix + the name, I think prefix will suffice.

Reply:
Ah, yes, you're right, sorry, hadn't read this correctly